### ISSUE TITLE
Update info on FAQ page

### ISSUE THREAD

**Issue by tasmie** [acm-chil/acm-chil-website#33]
https://www.chilconference.org/about.html

Under Live Interaction - "Poster Sessions" aren't mentioned anywhere else on the website or through recent email communications. I would suggest rewording to: 

"Q&A sessions with Keynote Speakers, tutorial authors and research paper authors, along with the virtual lunch socials, are all live via Zoom. These live sessions occur at specific times, so make sure you look at the Calendar to see what is live right now (see Schedule in menu)."

Under Getting Started - What if I have a problem? - we should specify an email address.... perhaps write: 

"If you have technical difficulties, please contact us by email at info@chilconference.org"

**Comment by corcra** [comment]
Implemented in https://github.com/acm-chil/acm-chil-website/commit/2b0dc3059ff37e07f234823483e24bf4522eea16
